learning_journal/__init__.py

```diff
@@ -0,0 +1,13 @@
+from pyramid.config import Configurator
+import os
+
+def main(global_config, **settings):
+    """ This function returns a Pyramid WSGI application.
+    """
+    settings['sqlalchemy.url'] = os.environ['DATABASE_URL']
```

Comment:
remember the cool trick we showed here to set a default value if one does not exist in the environment?